make/releasing.mk

```diff
@@ -41,7 +41,7 @@ release-dry-run: modvendor
 		-v `pwd`:/go/src/github.com/ovrclk/akash \
 		-w /go/src/github.com/ovrclk/akash \
 		troian/golang-cross:${GOLANG_CROSS_VERSION} \
-		--rm-dist --skip-validate --skip-publish
+		-f .goreleaser-docker.yaml --rm-dist --skip-validate --skip-publish
```

Comment:
`release-dry-run` and `release` targets should use default goreleaser config `.goreleaser.yaml`